draft-bormann-asdf-instance-information.md

```diff
@@ -211,16 +215,74 @@ Non-affordance:
 
 # Instance Information and SDF
 
-Instantiation doesn't produce an instance (ouch), which is the device,
-twin, etc., but a message.
+The instantiation of an SDF model does not produce an instance, which is, for example, a physical device or a digital twin.
+Instead, the instantiation produces an instance-related _message_, which adheres to a uniform message format and is always controlled by the corresponding SDF model.
+Depending on the recipient and the purpose of the message, different fields of the message format are present, reporting different kinds of information related to a Thing or causing it to change the state of the Thing when consumed by the recipient.
+Taking into account previous revisions of this document as well as {{-non-affordance}}, the potential use cases for instance-related messages indicate the following archetypes:
+
+1. *Context snapshots* that only report context information about a Thing
+2. *Proofshots* that report a Thing's state (or parts of it), which may include context information
+3. *Delta messages* that indicate state changes compared to a previous context snapshot or proofshot message
+4. *Patch messages* that cause a Thing's state to change
+5. *Identity manifests* that report information related to a Thing's identity
+6. *Construction messages* that initiate a Thing's (re)configuration or its comissioning
+
+These messages may be further clustered into the following three types:
+
+<!-- TODO: The names need to be improved -->
+
+1. *State reports*, both regarding affordances and context information (which includes information about a Thing's identity),
+2. *State report updates*, indicating changes that have occurred since a reference state report, and
+3. *State patches*, which include the original patch messages as well as construction messages
+
+The uniform message format can be used for all three message types.
+{{{#syntax}}} specifies the formal syntax of instance-related messages all normative statements as well as the examples in this document will adhere to.
+
+In the following, we will first outline a number of general principles for instance-related messages, before detailing the specific archetypes we define in this document.
+The specification text itself will be accompanied by examples that have been inspired by {{-non-affordance}} and {{-digital-twin}}.
+
+## Axioms for instance-related messages
 
-## Pre-structured types of messages
+<!-- TODO: Integrate this into the document in a better way -->
```

Comment:
I think for now capturing this is more important than nicely integrating it, so we may not want to tend to this TODO right away